src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
+
```

Comment:
- [ ] As shown in the project requirement below, you are currently missing the `checkStatus` action which you are required to create for the `categories` please kindly add this to your code base and submit for approval :+1: :sparkles: 
![cs](https://github.com/MPDADDY/bookstore/assets/89055502/f702814e-74cb-4ced-942b-1e451c1151d3)